components/src/dynamo/trtllm/workers/llm_worker.py

```
@@ -71,6 +71,37 @@
+    try:
+        if model_config.model_type not in _QWEN2_VL_FAMILY_MODEL_TYPES:
+            return None
+        vocab_size = getattr(model_config, "vocab_size", None)
+        if vocab_size is None:
+            vocab_size = getattr(
+                getattr(model_config, "text_config", None), "vocab_size", None
+            )
+        if vocab_size is None:
+            return None
+        return int(vocab_size) + 1
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)